umi_gripper/umi_gripper.xml

```diff
@@ -0,0 +1,172 @@
+<mujoco model="umi_gripper">
+  <compiler angle="radian" meshdir="assets" texturedir="assets" />
+
+  <option integrator="implicitfast" impratio="10" cone="elliptic" noslip_iterations="2">
+    <flag multiccd="enable" />
+  </option>
+
+  <asset>
+    <mesh name="base_link" file="base_link.stl" />
```

Comment:
BTW name is not necessary when it matches the file name. So you can remove it if you want a cleaner XML.